docs/applications/build-your-own/02-azure-app-service.md

```diff
@@ -0,0 +1,288 @@
+
+:::
+
+In the following example we compose all of the components of the Postgres authentication object into a `POSTGRES_URL` environment variable.
+
+```yaml title="Build a complex environment variable"
+app:
+  envs:
+    POSTGRES_URL: '@text "postgres://" + .connections.postgres.data.authentication.username + ":" + .connections.postgres.data.authentication.password + "@" + .connections.postgres.data.authentication.hostname + ":" + (.connections.postgres.data.authentication.port|tostring) + "/chat?sslmode=disable"'
+```
+#### Policy Examples
+
+Similarly to Environment Variables, IAM policies can also be mapped to your running application using JQ.
+
+Massdriver bundles will typically emit IAM Policies or Permissions information for their use case under the `security` field in the artifact. [Example](https://github.com/massdriver-cloud/artifact-definitions/blob/main/definitions/artifacts/postgresql-authentication.json#L28)
+
+For example: _if_ your application depended on an `Azure Storage Account` and you wanted to bind the `read/write` policy: 
+
+```yaml title="Binding IAM Policies to Cloud Workloads"
+  policies:
+    - .connections.azure_storage_account.data.security.iam.read/write
+```
+:::note
+
+Don't add this to your bundle without including Azure Storage Account as a connection dependency.
+
+:::
+
+A breakdown of the fields:
+
+* `.connections` - all connected infrastructure and applications.
+* `.connections.azure_storage_account` - the connected storage account (if required & present).
+* `.connections.azure_storage_account.data` - sensitive information in the storage account artifact.
+* `.connections.azure_storage_account.data.security` - security related information for the storage account.
+* `.connections.azure_storage_account.data.security.iam` - Principal of least privilege IAM policies exposed by the Azure storage account bundle.
+* `.connections.azure_storage_account.data.security.iam.read/write` - The `read/write` policy your application needs.
+
+#### Parameters
+
+The `params` in your `massdriver.yaml` are really good defaults for an Azure App Service deployment. Feel free to add or remove values depending on what you want to expose to your users / developers.
+
+#### Connections
+
+Your auto-generated connections:
+
+* `azure-service-principal` - the service principal used to deploy the application
+* `azure-virtual-network` - the virtual network your app will run on
+
+Additionally there are a few optional cloud authentication mechanisms. We suggest including these only if your application requires them:
+
+```yaml
+connections:
+  required:
+    - azure_authentication
+    - azure_network
+  properties:
+    azure_network:
+      $ref: massdriver/azure-virtual-network
+    azure_authentication:
+      $ref: massdriver/azure-service-principal
+    postgres:
+      $ref: massdriver/postgresql-authentication
+```
+
+## Publish
+
+At this point the application bundle can be published to the Massdriver bundle manager:
```

Comment:
```suggestion
At this point the application bundle can be published to the Massdriver package manager:
```

Reply:
Is it the `package manager`? I copy/pasta'd from the other section written by @coryodaniel

Reply:
the software which manages the storage of custom bundles is called a package manager. Think of it like npm in javascript or apt for linux.